cmake/UnifiedBuild.cmake

```diff
@@ -0,0 +1,174 @@
+# Options to control unified compilation. More information is available in the
+# documentation for `build_unified`.
+OPTION (UNIFIED_COMPILATION_ENABLED "Merge source files to speed up compilation" ON)
```

Comment:
can you please rename this to ENABLE_UNIFIED_COMPILATION? All our "enable" options start like this.

Reply:
OK, will do.